plugin/storage/cassandra/spanstore/reader.go

```diff
@@ -237,7 +237,10 @@ func (s *SpanReader) FindTraces(ctx context.Context, traceQuery *spanstore.Trace
 	}
 	if traceQuery.NumTraces == 0 {
 		traceQuery.NumTraces = defaultNumTraces
+	} else if traceQuery.NumTraces > 10000 {
+		traceQuery.NumTraces = 10000
```

Comment:
why this hardcoded upper limit? It should be user-configurable.

Reply:
^ This is fine. My bad, confused the two.